src/open_company_web/components/compensation.cljs

```diff
@@ -27,93 +28,149 @@
-              :label executives-label
-              :pluralize false}))
-          (when show-employees
-            (om/build report-editable-line {
-              :cursor comp-data
-              :key :employees
-              :prefix prefix
-              :label employees-label
-              :pluralize false}))
-          (when show-contrators
-            (om/build report-editable-line {
-              :cursor comp-data
-              :key :contractors
-              :prefix prefix
-              :label contractors-label
-              :pluralize false}))
-          (dom/div
-            (om/build report-line {
-              :prefix prefix
-              :number (thousands-separator total-compensation)
-              :label "total compensation this month"}))
-          (dom/div
-            (dom/span {:class "label"} "Report in: "
-              (dom/input {
-                :type "radio"
-                :name "report-type"
-                :value currency
-                :id "report-type-$"
-                :checked (not percentage)
-                :on-click #(handle-change comp-data false "percentage")})
-              (dom/label {:class "switch-vis" :for "report-type-$"} (str " " currency-symbol "  "))
-              (dom/input {
-                :type "radio"
-                :name "report-type"
-                :value "%"
-                :id "report-type-%"
-                :checked percentage
-                :on-click #(handle-change comp-data true "percentage")})
-              (dom/label {:class "switch-vis" :for "report-type-%"} "  Percent ")))
-          (om/build comment-component {:cursor comp-data :key :comment}))
-        (om/build pie-chart (get-chart-data comp-data head-data prefix))))))
+          currency-dict (utils/get-currency currency)
+          currency-symbol (utils/get-symbol-for-currency-code currency)
+          prefix (str currency-symbol " ")]
+      (p/panel {:header (dom/h3 "Compensation") :class "compensation clearfix"}
+        (dom/div {:class "compensation row"}
+          (dom/form {:class "form-horizontal col-sm-6"}
+
+            ;; Percentage
+            (dom/div {:class "form-group"}
+              (dom/label {:for "show-as" :class "col-md-4 control-label"} "Show as")
+              (dom/div {:class "btn-group btn-toggle col-md-3"}
+                (dom/button {
+                  :type "button"
+                  :class (str "btn btn-success" (when (not percentage) " active"))
+                  :on-click #(utils/handle-change comp-data false :percentage)}
+                  currency-symbol)
+                (dom/button {
+                  :type "button"
+                  :class (str "btn btn-default" (when percentage " active"))
+                  :on-click #(utils/handle-change comp-data true :percentage)}
+                  "%"))
+              (dom/p {:class "help-block"} (str "Viewers will see as " (if percentage "%" currency-symbol))))
+
+            ;; Founders
```

Comment:
@bago2k4  these sections, Founders, Execs, Employees are at a minimum big enough they should be moved into their own functions.

Ideally though, they aren't very DRY either. Should be moved into 1 function that DRYs them up.